fixtures/bucket_books_tree.py

```diff
@@ -39,13 +41,27 @@ def bucket_books_tree(s3_base_url, s3_queue_state_bucket_books):
         f"{s3_base_url}{s3_archive_folder}{i}@{j}.json" for i, j in zip(uuid_list, version_list)
     ]
 
-    book_tree = []
+    # list of approved book urls without http errors
+    approved_books_full_url_modified = []
+
+    for ublu in approved_books_full_url:
```

Comment:
This is really a comment on line 25 above (GH won't let me comment on that line, though 🤷‍♂️ ): Is that effectively de-duplicating by the `name` field? If so, won't that prevent checking multiple versions for a given book?

How many successful books this test currently finds for our recent staging pipeline? I believe it should be 85 successes and 28 failures (out of 113 books and 141 queue state entries).

Reply:
@rnathuji I see. Yes, we put this code there with Brenda to weed out duplicates but now the situation changed somewhat, so it is not necessary to find duplicate books (by title). I removed that bit of code, no more checking for duplicates, all books fall trough the filter :)